### ISSUE TITLE
[Bug]: Irregular implementation of kaiser in jax_lax_numpy

### ISSUE THREAD

**Issue by Enigmafiend** [unifyai/ivy#17987]
### Bug Explanation

Jax backend returns a array of 0s for testing occasionally even though the backend function seems stable enough not to. An array of 0s is mathmatically impossible for a correctly implemented window formula as the values as normalized.

### Steps to Reproduce Bug

Running the tf.test_signals.test_tensorflow_kaiser_window test as
# kaiser_window
@handle_frontend_test(
    fn_tree="tensorflow.signal.kaiser_window",
    dtype_and_window_length=helpers.dtype_and_values(
        available_dtypes=(["int32"]),
        min_dim_size=1,
        min_value=3,
        min_num_dims=1,
        max_value=20,
        max_dim_size=1,
        max_num_dims=1,
    ),
    dtype_and_beta=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"),
        min_dim_size=1,
        min_value=0,
        min_num_dims=1,
        max_value=80,
        max_dim_size=1,
        max_num_dims=1,
    ),
    test_with_out=st.just(False),
)
def test_tensorflow_kaiser_window(
    *,
    dtype_and_window_length,
    dtype_and_beta,
    frontend,
    test_flags,
    fn_tree,
    on_device,
):
    window_length_dtype, window_length = dtype_and_window_length
    beta_dtype, beta = dtype_and_beta
    print(f"window : {window_length} and beta : {beta}")
    helpers.test_frontend_function(
        input_dtypes=[window_length_dtype[0], beta_dtype[0]],
        frontend=frontend,
        test_flags=test_flags,
        fn_tree=fn_tree,
        on_device=on_device,
        window_length=window_length[0][0],
        beta=beta[0][0],
        rtol=1e-03,
        atol=1e-03,
    )

### Environment

Codespace VENV

### Ivy Version

Latest

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [ ] PyTorch
- [X] JAX

### Device

CPU

**Comment by Enigmafiend** [comment]
The reason is a hypothesis of why this bug occurs, this resultant out put only occurs with the hypothesis testing as far I can find but I am trying to find the proper combination that the test passes

**Comment by Enigmafiend** [comment]
![image](https://github.com/unifyai/ivy/assets/41777835/cfa6fbad-9ad8-4533-b698-a9681af4b97d)
